README.md

```diff
@@ -5,41 +5,87 @@
   </a>
 </p>
 
-# Docker Image
-    
-**1. Run a container with PHPSchool.**
-  
+# Getting Started
 
-```bash
+This process will provide you with a Docker environment where you can quickly run PHP School Workshops. We've gone ahead and pre-installed a few for you such as `learnyouphp`!
 
-  docker run -i -t -v ~/Dropbox/docker-phpschool:/phpschool/code:rw -e "PATH=/root/.php-school/bin:$PATH" rafaelcgstz/phpschool bash
+Once you get it up and running you should complete your tasks inside the `code` directory and use an editor such as [Atom](https://atom.io/) to write them.
 
-```
+Everything in the `code` directory will be mounted inside the Docker container at `/phpschool/code` so you can navigate there using the `cd` command and verify your solutions!
 
-**2. To verify your code.** 
-	
-In the docker container, navigate to `/phpschool` folder.
+## Docker Installation
 
-```bash
-  learnyouphp verify <name-of-solution-file>
+Head on over to [Docker](https://www.docker.com/products/docker) and download the latest version for your OS.
 
-```
-    
+#### Special Windows Instructions...
 
-**3. Advanced**
+Older Windows users may require [Docker Toolbox](https://www.docker.com/products/docker-toolbox) instead.
 
-This creates a docker container with the `learnyouphp`, `Callablefunctions` workshops and the `workshop-manager`.
+You will also need a better terminal capable of handling ANSI escape sequences, we recommend [ConEmu](https://github.com/Maximus5/ConEmu/releases)
 
-```bash
+## Downloading the Docker Image
+
+We recommend pulling this project down for the easiest way of getting started.
+
+You can do this in two different ways, either with `git` or manual download.
+
+With `git` just run `git clone git@github.com:php-school/docker-phpschool.git`
+
+Alternatively you can [download the files directly](https://github.com/php-school/docker-phpschool/archive/master.zip) and unzip them.
+
+## Starting the Docker Image
+
+Ensure your in your terminal and navigated to the directory of the files you just pulled down using the `cd` command.
```

Comment:
would be nice to have c+p commands here that followed on from the previous. The first command could be `cd ~/some-dir` then you can just repeat that in each section.